### ISSUE TITLE
Someone please upload the training set to google drive

### ISSUE THREAD

**Issue by DecentMakeover** [Canjie-Luo/MORAN_v2#41]
I have been training moran on my own set and it is giving me good results, but i was not able to download the training set from BaiduPan due to some issues, 

If anybody has been able to download the set,could they please upload it to google drive and share the link.

Thanks in advance.

**Comment by Canjie-Luo** [comment]
Sorry for that, I'll mark this issue as "good issue" to attract attention from others.

**Comment by DecentMakeover** [comment]
Thanks a lot for the support.Hope somebody uploads it!

**Comment by tnkong** [comment]
you can run this program to generate text image
https://github.com/Belval/TextRecognitionDataGenerator

**Comment by DecentMakeover** [comment]
Okay, let me check this, but are your sure the ground truth is also provided?

**Comment by tnkong** [comment]
there is serval option of data format, read run.py,  you can make each image_name by its label or generate a additional txt file to record "image_names + labels"

**Comment by DecentMakeover** [comment]
Okay, ill check this out.

**Comment by Canjie-Luo** [comment]
I just read the repo `README`. I am afraid the synthesized samples are not good enough. @DecentMakeover

**Comment by Canjie-Luo** [comment]
https://drive.google.com/open?id=1vfh6Gf-DLu5aJmf_FhpM27BImc5ObKjQ

**Comment by kanika02** [comment]
> I have been training moran on my own set and it is giving me good results, but i was not able to download the training set from BaiduPan due to some issues,
> 
> If anybody has been able to download the set,could they please upload it to google drive and share the link.
> 
> Thanks in advance.





hey! Can you please provide the pre-trained model of morgan (Can provide a link to the drive where it is uploaded). I am facing issues to train the model ,i need this pretrained model in my work .Please help me
